haskell-src/Concordium/Types/KonsensusV1.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |Types that are relevant only for endpoints exposed in ConsensusV1.
```

Comment:
Are you referring to the module name?

The module is called "KonsensusV1"  - because that is how it's named in the node repository. 
I.e. the types that are quite similar to the ones defined here.. (`Concordium/Types/KonsensusV1.hs`)

'ConsensusV1' is what we call the actual versioned parameter in `Parameters.hs`, i.e. `ConsensusVersion` in `Parameters.hs`.


:man_shrugging:  I can rename it to `ConsensusV1.hs` i.e. the module - I'm fine either way. @td-concordium got an opinion on this matter?

Reply:
I don't care enough to change naming now since it's obviously going to cause confusion elsewhere.

But it is confusing to have both K and C in various places for the same thing.